README.md

```diff
@@ -15,6 +15,10 @@ Our motivation is:
 - Simple installation
 - Multi-channel support 
 
+## Contribution
+
+This is an early draft for a possible replacement of pytorch/audio. At this stage, we encourage interested users to discuss specific proposals in the [issue section](https://github.com/keunwoochoi/torchaudio-contrib/issues). Further discussion primarily takes place in the pytorch #audio slack channel; please request an invitation at pytorch/audio#88 if you want to join.
```

Comment:
> Further discussion primarily takes place in the pytorch #audio slack channel;

I think Issues should be the first place to be hurdle-free. What do you think?

Reply:
yes I think you are right. lets try without slack for now. If it gets too messy we can revert that.